templates/robot_description/robot_moveit.launch.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2024, Stogl Robotics Consulting UG (haftungsbeschränkt)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Source of this file are templates in
+# [RosTeamWorkspace](https://github.com/StoglRobotics/ros_team_workspace) repository.
+#
+# Authors: Nikola Banović, Dr. Denis
+#
+import os
+import yaml
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import OpaqueFunction, DeclareLaunchArgument
+from launch.conditions import IfCondition, UnlessCondition
+from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+
+def load_yaml(package_name, file_path):
+    package_path = get_package_share_directory(package_name)
+    absolute_file_path = os.path.join(package_path, file_path)
+
+    try:
+        with open(absolute_file_path) as file:
+            return yaml.safe_load(file)
+    except OSError:  # parent of IOError, OSError *and* WindowsError where available
+        return None
+
+
+def launch_setup(context, *args, **kwargs):
+    use_sim_time = LaunchConfiguration("use_sim_time")
+    launch_rviz = LaunchConfiguration("launch_rviz")
+    only_rviz = LaunchConfiguration("only_rviz")
+
+    # Get SRDF via xacro
+    robot_srdf_content = Command(
+        [
+            PathJoinSubstitution([FindExecutable(name="xacro")]),
+            " ",
+            PathJoinSubstitution(
+                [FindPackageShare("$PKG_NAME$"), "srdf", "$ROBOT_NAME$.srdf.xacro"]
+            ),
+        ]
+    )
+    robot_description_semantic = {
+        "robot_description_semantic": robot_srdf_content.perform(context)
+    }
+
+    moveit_config_package_name = "$ROBOT_NAME$_moveit_config"
+
+    controllers_yaml = load_yaml(
+        moveit_config_package_name, "config/$ROBOT_NAME$/controllers.yaml"
```

Comment:
No we don't have it yet. We should do it. This can be done in the follow-up PR. I just wanted to have something that enables us to start working from somewhere. Making this setup is more or less copy-paste of the other setup scripts.

And, yes it would be great if you could do that!